strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/ProxyRepositoryProvider.java

```diff
@@ -129,4 +129,13 @@ protected ArtifactEntry provideArtirfactEntry(String storageId,
 
         return artifactEntry;
     }
+
+    @Override
+    public Path getPath(String storageId,
+                        String repositoryId,
+                        String artifactPath)
+            throws IOException
+    {
+        return null;
```

Comment:
`return null;`

Please `throw new UnsupportedOperationException()` instead of returning `null`